ontology/yaml/resources/SAFETY/entity_types/WTR.yaml

```diff
@@ -0,0 +1,39 @@
+# Copyright 2022 Google LLC

```

Comment:
Name of this file does not match the general type. You use`WSR` as the general type, not `WTR`. Please fix.

Reply:
i see, will do sir. thanks.